### ISSUE TITLE
"./perun-eth-demo help" returns error

### ISSUE THREAD

**Issue by matthiasgeihs** [perun-network/perun-eth-demo#15]
Running `./perun-eth-demo` lists `help` among the available commands, but running `./perun-eth-demo help` returns error `Error reading config file, open config.yaml: no such file or directory`.

**Comment by matthiasgeihs** [comment]
The info displayed when running `./perun-eth-demo` appears to be outdated:

> Usage: perun [command]

`perun` is not the name of demo. should be replaced by `perun-eth-demo`?

> Available Commands:
  demo        Two party payment Demo
  help        Help about any command

`help` command appears to be invalid as running it returns error.

**Comment by sebastianst** [comment]
> `perun` is not the name of demo. should be replaced by `perun-eth-demo`?
> 
I think that makes sense.

> `help` command appears to be invalid as running it returns error.

This sounds like it is caused by the main program always trying to load the config. Can probably easily be fixed by displaying the help and returning before loading the config.

**Comment by matthiasgeihs** [comment]
> This sounds like it is caused by the main program always trying to load the config. Can probably easily be fixed by displaying the help and returning before loading the config.

This is exactly what happened. My first idea was to load the config files only when running the demo command. However, for that I needed to make the config file parameters available to the package demo. Unfortunately, the demo command definition for some reason was in package cmd. To make it work, I refactored the demo command definition and its tests into package demo (where I think it belongs anyways).
